{"name": "test_modify_wiki_page_project_not_found", "status": "passed", "description": "Test modification fails due to invalid wiki identifier.", "attachments": [{"name": "log", "source": "650954af-c8fc-4966-a973-1d98b55886dc-attachment.txt", "type": "text/plain"}], "start": 1743499571069, "stop": 1743499571070, "uuid": "49ed4c61-260a-4cd8-9f71-5412ca272421", "historyId": "a3e22d7a758930edd2f6f676cfcecb24", "testCaseId": "a3e22d7a758930edd2f6f676cfcecb24", "fullName": "src.tests.ado.wiki.test_unit_ado_wiki.TestWikiApiWrapperNegative#test_modify_wiki_page_project_not_found", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "ado_wiki"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests.ado.wiki"}, {"name": "suite", "value": "test_unit_ado_wiki"}, {"name": "subSuite", "value": "TestWikiApiWrapperNegative"}, {"name": "host", "value": "fv-az1120-17"}, {"name": "thread", "value": "2592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.ado.wiki.test_unit_ado_wiki"}]}